etc/config/cuda.amazon.properties

```
@@ -442,8 +442,16 @@ libs.cueigen.versions.337.version=3.3.7
 libs.cueigen.versions.337.path=/opt/compiler-explorer/libs/eigen/v3.3.7
 libs.cueigen.versions.334.version=3.3.4
 libs.cueigen.versions.334.path=/opt/compiler-explorer/libs/eigen/v3.3.4
-libs.thrustcub.name=Thrust+CUB
-libs.thrustcub.description=CUDA collective and parallel algorithms
+libs.cccl.name=CUDA C++ Core libraries
+libs.cccl.description=Fundamental CUDA C++ libraries: Thrust, CUB, libcudacxx
+libs.cccl.versions=trunk:2002000
+libs.cccl.url=http://www.github.com/NVIDIA/cccl
```

This is probably a more "normalized" URL:
```suggestion
libs.cccl.url=https://github.com/NVIDIA/cccl
```